Dockerfile.j2

```diff
@@ -4,21 +4,23 @@
 # https://github.com/crate/docker-crate
 #
 
-FROM alpine:3.8
+FROM centos:7
 
-RUN addgroup -g 1000 -S crate \
-    && adduser -u 1000 -G crate -h /crate -S crate
+RUN groupadd crate && useradd -u 1000 -g crate -d /crate crate
+
+RUN curl --retry 8 -s {{ JDK_URL }} | tar -C /opt -zxf -
```

Comment:
Would it be possible, after downloading the JDK, to verify it via it's sha526 signature (i.e https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_linux-x64_bin.tar.gz.sha256) before you extract it?

Reply:
Not sure if there is a point in the verification if we retrieve the checksum from the same location? 
If java.net is compromised both files can be replaced. Curl/http is TCP based - so corruption due to transfers is not likely.

Reply:
Acht, yeah, good point.